packages/core/src/routes/language.ts

```diff
@@ -0,0 +1,25 @@
+import { Languages } from '@logto/schemas';
+
+import koaGuard from '@/middleware/koa-guard';
+import { findLanguageById } from '@/queries/language';
+
+import { AuthedRouter } from './types';
+
+export default function languageRoutes<T extends AuthedRouter>(router: T) {
```

Comment:
translations or i18n instead of language?

Reply:
per offline discussion: we will rename related names to `localization`